setup.cfg

```diff
@@ -81,6 +81,8 @@ metrics =
     sacrebleu~=2.2.1  # For disinformation_metrics, machine_translation_metrics
     langdetect~=1.0.9  # For ifeval_metrics
     immutabledict~=4.2.0  # For ifeval_metrics
+    gradio_client~=1.3  # For bigcodebench_metrics
+    tenacity~=9.0.0  # For bigcodebench_metrics
```

Comment:
any reason you use `tenacity` instead of `retrying` (which is already installed)?

Reply:
I did not notice that `retrying` has already been installed. Have switched to using `retrying`.